Added javax.faces-2.3.0-m09.jar and maven install plugin

shrinubabu opened the pull request:
Added javax.faces-2.3.0-m09.jar and maven install plugin as suggested by Rohit

shrinubabu committed:
changes to pom.xml to include jsf-core 2.3 dependency 

Able to build and run the application on internal tomcat after the
changes.

shrinubabu committed:
Added javax.faces-2.3.0-m09.jar and maven install plugin

jbuddha commented:
2.3.0-m09 can be added through pom as well, what is the need for adding jar file? 

```
<!-- https://mvnrepository.com/artifact/org.glassfish/javax.faces -->
<dependency>
    <groupId>org.glassfish</groupId>
    <artifactId>javax.faces</artifactId>
    <version>2.3.0-m09</version>
</dependency>
```

rohitvvv commented:
Just tested. My bad. The latest snapshots are available on maven. 
https://repo.maven.apache.org/maven2/org/glassfish/javax.faces/
Please continue to make the pom changes and don't use the jar.

shrinubabu committed:
changes to add jsf 2.3 from mavenrepository

Removed javax.faces-2.3.0-m09.jar from src/lib
Added dependency from mavenrepository as suggested by Buddha
Tested build and application is up and running with this.
